web-app/packages/lib/src/modules/project/components/ProjectPermissionsTemplate.vue

```diff
@@ -5,88 +5,109 @@ SPDX-License-Identifier: AGPL-3.0-only OR LicenseRef-MerginMaps-Commercial
-            <b v-bind="props">
-              {{ modelValue.username }}
-            </b>
+          <template #header>
+            <div class="w-11 grid grid-nogutter">
+              <!-- Visible on lg breakpoint > -->
+              <div
+                v-for="col in columns.filter((item) => !item.fixed)"
+                :class="['text-xs hidden lg:flex', `col-${col.cols ?? 4}`]"
+                :key="col.text"
+              >
+                {{ col.text }}
+              </div>
+              <!-- else -->
+              <div class="col-12 flex lg:hidden">Members</div>
+            </div>
           </template>
-          <span>
-            <span v-if="modelValue.profile.first_name">{{
-              modelValue.profile.first_name
-            }}</span>
-            <span v-if="modelValue.profile.last_name">
-              {{ modelValue.profile.last_name }}</span
-            >
-          </span>
-        </v-tooltip>
-        <b v-else>
-          {{ modelValue.username }}
-        </b>
-      </template>
-
-      <template #item.permissions="{ item }">
-        <v-select
-          :model-value="actualPermissions(item)"
-          :items="permissionStates"
-          @update:model-value="(e) => valueChanged(item, e)"
-          :disabled="!isProjectOwner"
-          hide-details
-          label="reader"
-          single-line
-          class="input-selection"
-          style="width: 120px"
-        >
-        </v-select>
-      </template>
 
-      <template #item.remove="{ item }">
-        <div class="justify-center px-0">
-          <v-btn
-            :disabled="!canRemoveUser(item.user.id)"
-            @click="removeUser(item.user)"
-            icon
-          >
-            <v-icon color="red darken-3">delete</v-icon>
-          </v-btn>
-        </div>
-      </template>
-    </v-data-table>
-    <button
-      ref="hidden-input"
-      id="change-permissions-input"
-      style="visibility: hidden"
-    />
-  </v-layout>
+          <!-- table rows -->
+          <template #list="slotProps">
+            <div
+              v-for="item in slotProps.items"
+              :key="item.id"
+              class="flex align-items-center hover:bg-gray-50 border-bottom-1 border-gray-200 text-sm px-4 py-2 mt-0"
+            >
+              <div class="w-11 grid grid-nogutter">
+                <!-- Columns, we are using data view instead table, it is better handling of responsive state -->
+                <div
+                  v-for="col in columns.filter((item) => !item.fixed)"
+                  :key="col.value"
+                  :class="[
+                    'flex flex-column justify-content-center col-12',
+                    `lg:col-${col.cols ?? 4}`
+                  ]"
+                >
+                  <p class="text-xs opacity-80 mb-1 lg:hidden">
+                    {{ col.text }}
+                  </p>
+                  <span :class="col.textClass">
+                    <PAvatar
```

Comment:
do we need avatar placeholder (we do not really support them at the moment)?

Reply:
We are creating avatars description from first letter of username without changing color of avatar.

Reply:
We can clean avatars in next PRs.